docker/pool/builder/lib/builder/docker.rb

```diff
@@ -4,17 +4,21 @@
 module Builder
   module Docker
     CONTAINER_PORT = 80
+    ::Docker.options[:read_timeout] = 15 * 60 # 15 minutes
 
+    attr_accessor :logger
+    module_function :logger, :logger=
```

Comment:
@ainoya It seems that `docker.rb`assumes that it can access logger. If so, it may be better to make modules consistently:

* get logger as parameters
* not to get logger as parameters and always use `attr_accessor` and `module_function`